plugins/catalog-backend/src/generated/apis/DefaultApi.server.ts

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// ******************************************************************
+// * THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY. *
+// ******************************************************************
+import type {
+  AnalyzeLocationRequest,
+  AnalyzeLocationResponse,
+  CreateLocation201Response,
+  CreateLocationRequest,
+  EntitiesBatchResponse,
+  EntitiesQueryResponse,
+  Entity,
+  EntityAncestryResponse,
+  EntityFacetsResponse,
+  GetEntitiesByRefsRequest,
+  GetLocations200ResponseInner,
+  Location,
+  RefreshEntityRequest,
+  ValidateEntityRequest,
+} from '@backstage/catalog-client/alpha';
```

Comment:
Tbh for pure client packages it feels like it _should_ be fine to export all types from top level, might get really messy for `-common` packages though, which is what we generally wanna stick to. Wonder if `*-common/client` could be an alright pattern?

Btw, is the server client import option missing from the script in this PR or how does it know to import from `/alpha`?

Reply:
I like `*-common/client` as a pattern, would probably want a codemod to adjust the package.json for multiple entry points 😬 , but overall that makes sense.

> Is the server client import option missing/

Yup, working on that right now 😅

Reply:
Updated to use `-common/client`!